src/components/ImageSlider/Arrow.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import { View } from 'react-native'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+class Arrow extends Component {
+  render() {
+    const {
+      name,
+      color,
+      enableBackground,
+      backgroundColor,
+      backgroundRounding,
+      enableBorder,
+      borderSize,
+      borderColor,
+      size,
+      onPress,
+      style,
+    } = this.props
+    const viewHeight =
```

Comment:
Can you add some spacing around these multiline const statements so it's more readable?